db/migrate/20170218190259_add_zip_code_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddZipCodeToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :zip_code, :integer
```

Comment:
zip codes aren't actually scalars.  Storing as an integer will truncade leading zeros for places like Boston (02108 would be stored as 2108)